x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/action_bar/action_bar.tsx

```diff
@@ -139,11 +146,23 @@ export const ActionBar = ({
             <EuiFlexItem grow={false}>
               <WarningText>{!isValid && hasBeenSubmitted && VALIDATION_ERROR_LABEL}</WarningText>
             </EuiFlexItem>
+
+            <EuiToolTip
+              position="top"
+              content={<p>Works on any kind of element &mdash; buttons, inputs, you name it!</p>}
```

Comment:
What content is this supposed to have?

Reply:
Looking at this further, I think this whole code block isn't supposed to be here.